compiler/ast.nim

```diff
@@ -722,7 +722,7 @@ type
     mInstantiationInfo, mGetTypeInfo, mGetTypeInfoV2,
     mNimvm, mIntDefine, mStrDefine, mBoolDefine, mGenericDefine, mRunnableExamples,
     mException, mBuiltinType, mSymOwner, mUncheckedArray, mGetImplTransf,
-    mSymIsInstantiationOf, mNodeId, mPrivateAccess, mZeroDefault
+    mSymIsInstantiationOf, mNodeId, mPrivateAccess, mZeroDefault, mSameSeqPayload
```

Comment:
Does not require a magic, a codegen proc is good enough.

Reply:
=> https://github.com/nim-lang/Nim/pull/22671